var/spack/repos/builtin/packages/py-astropy/package.py

```diff
@@ -25,42 +27,75 @@ class PyAstropy(PythonPackage):
     variant('extras', default=False, description='Enable extra functionality')
 
     # Required dependencies
+    depends_on('python@3.8:', when='@5.0:', type=('build', 'run'))
+    depends_on('python@3.7:', when='@4.2:', type=('build', 'run'))
     depends_on('python@3.6:', when='@4.0:', type=('build', 'run'))
     depends_on('python@3.5:', when='@3.0:', type=('build', 'run'))
     depends_on('python@2.7:2.8,3.4:', when='@2.0:', type=('build', 'run'))
     depends_on('python@2.7:2.8,3.3:', when='@1.2:', type=('build', 'run'))
     depends_on('python@2.6:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm@6.2:', when='@5.0:', type='build')
+    depends_on('py-setuptools-scm', when='@4.1:', type='build')
+    depends_on('py-cython@0.29.30:', when='@5.1:', type='build')
+    depends_on('py-cython@0.29.22:', when='@4.2.1:', type='build')
+    depends_on('py-cython@0.29.14:', when='@4.0.2:', type='build')
     depends_on('py-cython@0.29.13:', type='build')
+    depends_on('py-jinja2@3.0.3:', when='@5.0.3:5.0.4', type='build')
+    depends_on('py-jinja2@2.10.3:', when='@4.0.2:', type='build')
+    depends_on('py-numpy@1.18:', when='@5.0:', type=('build', 'run'))
+    depends_on('py-numpy@1.17:', when='@4.2:', type=('build', 'run'))
     depends_on('py-numpy@1.16:', when='@4.0:', type=('build', 'run'))
     depends_on('py-numpy@1.13:', when='@3.1:', type=('build', 'run'))
     depends_on('py-numpy@1.10:', when='@3.0:', type=('build', 'run'))
     depends_on('py-numpy@1.9:', when='@2.0:', type=('build', 'run'))
     depends_on('py-numpy@1.7:', when='@1.2:', type=('build', 'run'))
     depends_on('py-numpy', type=('build', 'run'))
     depends_on('pkgconfig', type='build')
+    depends_on('py-packaging@19.0:', when='@5.0:', type='build')
+    depends_on('py-extension-helpers', when='@4.1:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-extension-helpers', when='@4.1:', type='build')
```
I'm guessing, it would be good to confirm if it is needed to import anything at run-time.